src/Npgsql.NodaTime/IntervalHandler.cs

```diff
@@ -10,26 +10,14 @@ public class IntervalHandlerFactory : NpgsqlTypeHandlerFactory<Period>
     {
         // Check for the legacy floating point timestamps feature
         protected override NpgsqlTypeHandler<Period> Create(NpgsqlConnection conn)
-            => new IntervalHandler(conn.HasIntegerDateTimes);
+            => new IntervalHandler();
```

Comment:
I think that even before floating point support was added by CrateDB, we still checked the HasIntegerDateTime is true, and threw an exception otherwise. This means that if for any reason someone happens to run on an old version somehow, they get an informative exception instead of corrupted data.

It seems like a good idea to go back to that, it doesn't seem to cost us anything (if interested you can take a look at some version before the CrateDB support).